doc/source/tutorials/Q-Factor.ipynb

```diff
@@ -0,0 +1,464 @@
+    "\n",
+    "## Example with a Parallel RLC Circuit\n",
+    "To illustrate the usage of the `Qfactor` class of `scikit-rf`, a parallel RLC circuit as illustrated above is used as an canonical example for which analytical formulas are available for benchmarking.\n",
+    "\n",
+    "<img src=\"figures/Parallel_RLC_resonator.svg\" width=\"300\"/>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b361222e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import skrf as rf\n",
+    "import matplotlib.pyplot as plt\n",
+    "rf.stylely()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1735918b",
+   "metadata": {},
+   "source": [
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb7eba12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "C = 1e-6  # F\n",
+    "L = 1e-9  # H\n",
+    "R = 30  # Ohm\n",
+    "Z0 = 50  # Ohm\n",
+    "\n",
+    "freq = rf.Frequency(5, 5.2, npoints=501, unit='MHz')\n",
+    "media = rf.DefinedGammaZ0(frequency=freq, z0=Z0)  # ideal line (no loss)\n",
+    "random_d = np.random.uniform(-np.pi, np.pi)  # a random length for the sake of the example\n",
+    "\n",
+    "resonator = media.line(d=random_d, unit='rad') \\\n",
+    "                **media.shunt_inductor(L) ** media.shunt_capacitor(C) \\\n",
+    "                ** media.shunt(media.resistor(R)**media.short()) ** media.open()\n",
+    "\n",
+    "resonator.plot_s_db()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b291cfb1",
+   "metadata": {},
+   "source": [
+    "Analytical formulas for this case are available and given by [2,3]:\n",
+    "$$ f_{\\mathrm{res}} = \\frac{1}{2\\pi \\sqrt{L C}} $$\n",
+    "$$ Q = \\omega_r R C = \\frac{R C}{\\sqrt{L C}}$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c099dff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def f_res_RLC(L, C):\n",
+    "    return 1/(2*np.pi*np.sqrt(L*C))\n",
+    "\n",
+    "def Q_RLC(R, L, C):\n",
+    "    return R * C / np.sqrt(L*C)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99d749a3",
+   "metadata": {},
+   "source": [
+    "he unloaded Q-factor $Q_0$ is a characteristic of the resonator itself, in the absence of any loading effects caused by the external circuitry. In practice, of course, the resonator is connected and coupled to this circuitry which have the effect to lower the Q-factor of the loaded circuit $Q_L$. \n",
+    "\n",
+    "Hence, when the resonator is connected to the transmission line, it is loaded with an equivalent series resistance $Z0$ (assuming no loss in the transmission line), while it is not the case for the resonator alone:"
```

Comment:
Maybe to make it even more clear:
"... it is loaded with the port impedance $Z0$, which is connected in parallel to the intrinsic resonator resistor $R$. The equivalent resonator load is therefore $R||Z0 = (R*Z0) / (R+Z0)$.